.vscode/launch.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Test262 (subset in launch.json)",
+      "request": "launch",
+      "type": "node",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run", "test262"],
+      "protocol": "inspector",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": false,
```

Comment:
What does this do?

Reply:
Cool, we have an [answer from VSCode](https://github.com/microsoft/vscode-js-debug/issues/1102#issuecomment-917106779): `protocol` and  `disableOptimisticBPs` are not used anymore. I'll remove them. The others look good.